lib/tesla/middleware/keep_request.ex

```diff
@@ -1,6 +1,6 @@
 defmodule Tesla.Middleware.KeepRequest do
   @moduledoc """
-  Store request body & headers into opts.
+  Store request `body` and `headers` into opts.
```

Comment:
I think "body" and "headers" here are referring the HTTP terms, not fields in the struct. However, `opts` is a field.

What about this? 

> Keeps request body and headers in `opts`.